source/docs/03-metadata.xml

```diff
@@ -1382,156 +1382,318 @@
+                     <p>Detailed analysis of the title page and other preliminaries of older printed books and manuscripts is of major importance in descriptive bibliography and the cataloging of printed books. The following elements are suggested as a means of encoding the major features of most title pages for faithful rendition:</p>
                      <p>
                         <specList>
-                           <specDesc key="att.foliumSurfaces" atts="recto"/>
-                           <specDesc key="att.foliumSurfaces" atts="verso"/>
+                           <specDesc key="titlePage"/>
+                           <specDesc key="p"/>
+                           <specDesc key="table"/>
+                           <specDesc key="list"/>
+                           <specDesc key="quote"/>
+                           <specDesc key="lg"/>
                         </specList>
                      </p>
-                     <p>These attributes are used to point to the <att>xml:id</att> of a <gi scheme="MEI">surface</gi> element.</p>
+                     <p>The following example shows the encoding of the title page of Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi>. Note the use of the <gi scheme="MEI">lb</gi> element to mark the line breaks present in the original.</p>
                      <p>
                         <figure>
-                           <head>Positional attributes on folium and bifolium</head>
-                           <graphic url="../images/ExampleImages/foliadesc-02.png"/>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/text/text-sample371.txt" parse="text"/></egXML>
                         </figure>
                      </p>
-                     <p>On <gi scheme="MEI">bifolium</gi>, the corresponding attributes are:</p>
+                     <p>The physical rendition of title page information is often of considerable importance. One approach to this requirement would be to use the <gi scheme="MEI">rend</gi> element, described in chapter <ptr target="#sharedTextRendition"/> to specify the rendition of each of the components of the title page. Another would be to employ a CSS stylesheet. Finally, a module customized for the description of typographic entities such as pages, lines, rules, etc., bearing special-purpose attributes to describe line-height, leading, degree of kerning, font, etc. could be employed.</p>
```

Comment:
```suggestion
                     <p>The physical appearance of title page information is often of considerable importance. One approach to capturing the appearance is be to use the <gi scheme="MEI">rend</gi> element, described in chapter <ptr target="#sharedTextRendition"/> to specify the placement of each of the components of the title page. Another would be to employ a CSS stylesheet. Finally, a module customized for the description of typographic entities such as pages, lines, rules, etc., bearing special-purpose attributes to describe line-height, leading, degree of kerning, font, etc. could be employed.</p>
```

Reply:
"Rendition" isn't really the right word to use here. I have substituted "appearance" since I think that is what is meant. 

I would also scrap the sentence "Another would be to employ a CSS stylesheet" unless you are prepared to describe how to do that in MEI. 😓

Reply:
+1 for these changes, thanks @ahankinson. Just one note: the "be" after "capturing the appearance is" has to be removed.

Reply:
Fixed.

Reply:
resolved manually